src/chi/components/picker/picker.scss

```diff
@@ -14,6 +14,124 @@ $sizes: (
   )
 );
 
+.m-picker {
+  input {
+    &[type='checkbox'],
+    &[type='radio'] {
+      &.a-picker {
+        & + label {
+          span.a-picker__checkbox {
+            border: 2px solid set-color(grey, 50);
+            border-radius: 3px;
+            color: set-color(grey, 50);
+            height: 1rem;
+            position: relative;
+            width: 1rem;
+
+            &::before {
+              border: 2px solid set-color(grey, 50);
+              border-left: 0;
+              border-top: 0;
+              content: '';
+              display: block;
+              height: 9px;
+              left: calc(50%-0.156rem);
+              position: absolute;
+              top: calc(50%-0.37rem);
+              transform: rotate(45deg) scale(0);
+              transition-duration: 0.2s;
+              width: 5px;
+            }
+          }
+
+          span.a-picker__radio {
+            border: 2px solid set-color(grey, 50);
+            border-radius: 0.5rem;
+            color: set-color(grey, 50);
+            height: 1rem;
+            position: relative;
+            width: 1rem;
+
+            &::before {
+              background-color: set-color(grey, 50);
+              border-radius: 0.25rem;
+              content: '';
+              height: 0.5rem;
+              left: calc(50%-0.25rem);
+              position: absolute;
+              top: calc(50%-0.25rem);
+              transform: scale(0);
+              transition-duration: 0.2s;
+              width: 0.5rem;
+            }
+          }
+
+          & .a-picker__label {
+            margin-left: 0.5rem;
+          }
+
+          &:hover {
+            span.a-picker__checkbox {
+              &::before {
+                transform: rotate(45deg) scale(1);
+              }
+            }
+            
+            span.a-picker__radio {
+              &::before {
+                transform: scale(1);
+              }
+            }
+          }
+
+          .m-picker__content {
+            &--start {
+              & span.a-picker__label {
+                  padding-left: 0.5rem;
+              }
+
+              & .a-picker__description {
+                color: $text-color;
+                font-size: $text-small;
+                font-weight: 400;
+                line-height: 1.25rem;
+                margin-left: 2.5rem;
+                padding: 0.5rem 0;
+              }
+            }
+          }
+        }
+      }
+
+      &:checked {
+        & + label {
+          span.a-picker__checkbox {
+            background-color: $brand-color;
+            border: 2px solid $brand-color;
+            color: $brand-color;
+  
+            &::before {
```

Comment:
```
              border: 2px solid solid set-color(white);
              border-right: none;
              border-top: none;
              transform: rotate(-45deg) scale(1);
```

Reply:
All done. Could you check out the Sizes examples of the Picker group docs page? I had to remove `.-d--flex.-flex--wrap` between legend and m-pickerGroup and it affected vertical alignment between the pickers that are in one row.

Reply:
All done, thanks for the suggestions. There is one thing about Picker group page Sizes examples. For each size, we have 2 picker groups in a row. In order to align them well vertically, we need `.-d--flex.-flex--wrap`. So m-pickerGroup and Leged are not siblings. So I had to add `-mb--1` to the legend to achieve the same spacing.